README.md

```diff
@@ -1,27 +1,168 @@
-# Role Name
+# Gitlab
 
-A brief description of the role goes here.
+This role configures and installs a gitlab instance.
```

Comment:
The software is called GitLab: https://www.wikiwand.com/en/GitLab

Reply:
fixed